README.md

```diff
@@ -6,17 +6,23 @@
 
 https://stashapp.cc
 
-**Stash is a Go app which organizes and serves your porn.**
+**Stash is a locally hosted web-based app written in Go which organizes and serves your porn.**
 
-See a demo [here](https://vimeo.com/275537038) (password is stashapp).
+* It can gather information about movies in your collection from the internet, and is extensible through the use of community-built plugins.
```

Comment:
Prefer `videos` rather than `movies` because we also have movies and don't want to conflate the two concepts.